README.md

```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
 Create a `.commitlintrc.json` file and extend the default recommended config like so:
 ```js
 {
-  "extends": ["@biotope/quality-gate/config/.commitlintrc.js"]
+  "extends": ["@virtualidentityag/quality-gate/config/.commitlintrc.js"]
   // Add any other definition here.
 }
 ```
 
+## Recommended settings
+### Stencil
+#### .eslintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.eslintrc.js",
+  "rules": {
+    "class-methods-use-this": 0,
+    "no-script-url": 0,
+    "import/no-default-export": 1,
+    "react/prop-types": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "react/jsx-key": 0,
+    "no-use-before-define": 0
+  },
+  "plugins": [
+    "eslint-plugin-local-rules"
+  ]
+}
+```
+#### .stylelintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.stylelintrc.js",
+  "rules": {
+    "selector-class-pattern": ".",
+    "selector-no-qualifying-type": null,
+    "selector-type-no-unknown": [ true, { "ignore": ["custom-elements"] } ],
```

Comment:
is party of quality gate

Reply:
We (UI) should actually enable it again, to support with versioning

Reply:
Okay, removed it here.